content/04kapitel.tex

```diff
@@ -743,6 +852,73 @@ \subsubsection{SKIOS-80 Article View Page}
 Bearbeitet von Simong Morgenstern und Jonas Eppard.
 Reviewed von Lukas Huida.
 
+\subsubsection{SKIOS-78 Subscription Page}
+Das Ticket \enquote{SKIOS-78 Subscription Page} hatte folgende User Story:
+\begin{quotation}
+    As a user of Skiosa I would like to have a page for my subscriptions so that I can use them.
+
+    Goal of this story is to use the backend components provided in SKIOS-77 and to create a subscription page following our design.
+
+    Note: Subscription indicator of mockup will not be in scope for this story.
+\textbf{Acceptance Criteria}
+\begin{itemize}
+    \item Subcription page follows design of figma mockups
+    \item Page contains list of the current users subscriptions
+    \item For non logged in users, this page redirects to login
+\end{itemize}
+\end{quotation}
+Bearbeitet von Amos Groß.
+Reviewed von Lukas Huida.
+
+\subsubsection{SKIOS-94 Overview Page}
+Das Ticket \enquote{SKIOS-94 Overview Page} hatte folgende User Story:
+\begin{quotation}
+    As a user I would like to have a homepage.
+
+    Goal of this story is to use the endpoints of the preliminary recomendation service and to display them as seen in the mockup.
+
+    The recomended feeds are not to be regarded here.
+\textbf{Acceptance Criteria}
+\begin{itemize}
+    \item Overview Page exists at root ('/')
+    \item Overview Page follows design scheme 
+\end{itemize}
+\end{quotation}
+Bearbeitet von Amos Groß.
+Reviewed von Simon Morgenstern.
+
+\subsubsection{SKIOS-114 Roadmap in Confluence}
+Das Ticket \enquote{SKIOS-114 Roadmap in Confluence} hatte folgende User Story:
+\begin{quotation}
+    As a Product owner I would like a roadmap in confluence.
+\textbf{Acceptance Criteria}
+\begin{itemize}
+    \item Preliminary Roadmap in Front Page of Confluence is Replaced
+    \item Roadmap reflects or PIs
+\end{itemize}
+\end{quotation}
+Bearbeitet von Amos Groß.
+Reviewed von Lukas Huida.
+
+\subsubsection{SKIOS-123 Latex Requirements Page}
+Das Ticket \enquote{SKIOS-123 Latex Requirements Page} hatte folgende User Story:
+\begin{quotation}
+    As a team member I would like to have our requirements documented in confluence so that I won’t fail this class.
```

Comment:
```suggestion
    As a team member I would like to have our requirements documented in confluence so that I won’t fail this class.
```
```suggestion
    As a team member I would like to have our requirements documented in confluence so that I won’t fail this class. \\
```